Cake.Recipe/Content/build.cake

```diff
@@ -398,35 +398,95 @@ public void CopyBuildOutput(BuildVersion buildVersion)
         if (parsedProject.IsLibrary() && (parsedProject.HasPackage("xunit") || parsedProject.HasReference("xunit.core")))
         {
             Information("Project has an output type of library and is an xUnit Test Project: {0}", parsedProject.RootNameSpace);
-            var outputFolder = BuildParameters.Paths.Directories.PublishedxUnitTests.Combine(parsedProject.RootNameSpace);
-            EnsureDirectoryExists(outputFolder);
-            CopyFiles(GetFiles(parsedProject.OutputPath.FullPath + "/**/*"), outputFolder, true);
-            continue;
+
+            if (parsedProject.IsVS2017ProjectFormat)
```

Comment:
We may perhaps think about refactoring this whole if/else statements to a helper method, instead of duplicating the effort multiple times.

Reply:
There is already quite a bit of duplication in this `CopyBuildOutput` method, so it would be quite a large undertaking to refactor it.  Don't have the energy to do it just now though 😄 Personally happy to add this duplication, with a view to fix this entire method at a later date.

Reply:
Let me give it a shot then, before we merge this in.

Reply:
You are my hero!